docs/source/security/cors.mdx

```diff
@@ -77,22 +80,24 @@ If you create an API on the public internet to serve resources to your _own_ web
 
 #### Public or embedded APIs
 
-If you create a public API or an API to embed in websites you don't control yourself, you probably want to allow _all_ origins to access your server's resources. You can use Apollo Server's default ACAO value (the wildcard `*`) in these situations, allowing requests from any origin.
+If you create a public API or an API to embed in websites you don't control yourself, you probably want to allow _all_ origins to access your server's resources. You can use the [`startStandaloneServer`]('s ACAO value (the wildcard `*`) in these situations, allowing requests from any origin.
```

Comment:
this markdown looks wrong, there's like half a link here

Reply:
Oh my bad, I think I accidentally snipped it when copying and pasting. Fixed!